File: py/IEC61968/InfIEC61968/InfERPSupport/ErpPayableLineItem.py
from IEC61968.Common.Status import Status
from IEC61968.InfIEC61968.InfERPSupport.ErpIdentifiedObject import ErpIdentifiedObject
from IEC61968.InfIEC61968.InfERPSupport.ErpJournalEntry import ErpJournalEntry
from IEC61968.InfIEC61968.InfERPSupport.ErpPayable import ErpPayable
from IEC61968.InfIEC61968.InfERPSupport.ErpPayment import ErpPayment


class ErpPayableLineItem(ErpIdentifiedObject):

    def __init__(self):
        super().__init__()
        self.status = Status()
        self.erp_journal_entries = ErpJournalEntry()
        self.erp_payable = ErpPayable()
        self.erp_payments = ErpPayment()

File: java/IEC61968/InfIEC61968/InfERPSupport/ErpPayableLineItem.java
import IEC61968.Common.Status;

/**
 * Of an ErpPayable, a line item references an ErpInvoiceLineitem or other source
 * such as credit memos.
 * @created 25-Dec-2023 8:45:21 PM
 */
public class ErpPayableLineItem extends ErpIdentifiedObject {

	public Status status;
	public ErpJournalEntry ErpJournalEntries;
	public ErpPayable ErpPayable;
	public ErpPayment ErpPayments;

	public ErpPayableLineItem(){

	}
}//end ErpPayableLineItem